src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContainerAppsResources/getResourceHelpers.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ContainerAppsAPIClient, ManagedEnvironment } from "@azure/arm-appcontainers";
+import type { ResourceGroup } from "@azure/arm-resources";
+import { ResourceGroupListStep, getResourceGroupFromId, uiUtils } from "@microsoft/vscode-azext-azureutils";
+import { ISubscriptionActionContext, nonNullProp } from "@microsoft/vscode-azext-utils";
+import type { ContainerAppModel } from "../../../../tree/ContainerAppItem";
+import { createContainerAppsAPIClient } from "../../../../utils/azureClients";
+import type { DefaultContainerAppsResources } from "./getDefaultContainerAppsResources";
+
+/**
+ * @throws May throw an error if there are any issues with the associated network calls
```

Comment:
Honestly, we probably don't need this disclaimer. It's an inherent risk of every SDK call we make, even listing the resources on activation.

That being said, we _may_ want to try/catch only because we can technically still run the create command if we return everything as `undefined`.